plugins/outputs/groundwork/groundwork.go

```diff
@@ -254,12 +260,18 @@ func (g *Groundwork) parseMetric(metric telegraf.Metric) (string, *transit.Dynam
 		if err != nil {
 			return "", nil, err
 		}
+		if typedValue.ValueType == transit.StringType {
+			g.Log.Warn("string values are not supported, skipping")
+			continue
+		}
 
 		serviceObject.Metrics = append(serviceObject.Metrics, transit.TimeSeries{
 			MetricName: value.Key,
 			SampleType: transit.Value,
 			Interval: &transit.TimeInterval{
 				EndTime: lastCheckTime,
+				// Temporary work around to avoid error from the server.
+				StartTime: transit.NewTimestamp(),
```

Comment:
Same here, would this be accepted?
```suggestion
				StartTime: lastCheckTime,
```

Reply:
Fixed